fe/estimator.py

```diff
@@ -99,29 +136,41 @@ def _deltaG(model, sys_params):
 
         results = [x.result() for x in futures]
 
-    du_dls = []
+    if not (callback is None):
```

Comment:
easier to read for me would be:
```
if callback is not None
```
(less parentheses)

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/360/commits/ed646fbccbe0f70dc6244fd8c16df8feb95d368c